app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -360,6 +378,7 @@ export default {
   },
   created() {
     this.getStatus();
+    this.setRecipientEmail();
```

Comment:
Consider having this method only being invoked when the user selects revising state instead of running it all the time on page/component load to reduce computation and network calls.